catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java

```diff
@@ -76,16 +76,15 @@
 	private CategoryListAdapter adapter;
 
 	private static final int[] OBJECT_GENERAL_PROPERTIES_ITEMS = { R.string.formula_editor_object_transparency,
-			R.string.formula_editor_object_brightness, R.string.formula_editor_object_color/*,
-			R.string.formula_editor_object_distance_to*/ };
```

Comment:
Just to be sure, the `distance to` feature hasn't been enabled in any release yet, right?
So we don't have to cover backward compatibility.